src/main/java/lotto/domain/lotto/Lotto.java

```diff
@@ -0,0 +1,27 @@
+package lotto.domain.lotto;
+
+public class Lotto {
+  public static final Integer PRICE = 1000;
+  private final LottoNumbers value;
+
+  public Lotto(LottoNumbers value) {
+    this.value = value;
+  }
+
+  public Lotto() {
+    this.value = LottoNumbers.valueOf();
```

Comment:
```suggestion
    this(LottoNumbers.valueOf());
```
하나의 생성자에 검증 로직이라던지 공통적인 코드를 넣어두고 
다른 생성자나 정적 팩토리 메서드에서 활용하는 편이 중복을 제거하는데에 용이합니다.